src/tasks/hub/dependencies.js

```diff
@@ -1,7 +1,16 @@
 const semver = require('semver')
 const { Logger } = require('KegLog')
 const { getHubRepos } = require('KegUtils/hub/getHubRepos')
-const { reduceObj, mapObj, get, set } = require('@keg-hub/jsutils')
+const { reduceObj, mapObj, get, set, isEmptyColl } = require('@keg-hub/jsutils')
```

Comment:
on test #2, it mentions that there should be `No mismatched dependency versions found!` when running `keg hub deps`

but I see 1 mismatch when I run the cmd

-[image](https://user-images.githubusercontent.com/3317835/111512586-229d7080-870d-11eb-9cb2-a6ba40c6aca8.png)

Reply:
i switched to the wrong branch. this is fine